contents/docs/self-host/deploy/gcp.mdx

```diff
@@ -58,3 +58,62 @@ As a troubleshooting tool, you can allow HTTP access by setting `ingress.gcp.for
 
 ## Uninstalling the chart
 <UninstallingSnippet />
+
+## Clickhouse Configuration
+
+By default clickhouse is provisioned with standard gcp persistent disk. If you want to specify your own persistent volume claim or switch to a different type of disk
+you can specify the volume claim within `values.yaml`.
```

Comment:
Maybe add a sentence in the end bringing an example of when one would want to do this, you ran into scaling problems, right?